docs/source/intro.rst

```diff
@@ -25,9 +25,9 @@ of each implementation with the generic usage that Dask demanded. People may fin
 `code`_ which parses URLs and creates file-system
 instances interesting.
 
-.. _Dask: http://dask.pydata.org/en/latest/
-.. _docs: http://dask.pydata.org/en/latest/remote-data-services.html
-.. _code: https://github.com/dask/dask/blob/master/dask/bytes/core.py#L266
+.. _Dask: https://dask.pydata.org/en/latest/
+.. _docs: https://dask.pydata.org/en/latest/how-to/connect-to-remote-data.html
+.. _code: https://github.com/fsspec/filesystem_spec/blob/4f0eb48/fsspec/core.py#L525
```

Comment:
Note that https://github.com/dask/dask/commit/7e4beffb is the last [`dask/bytes/core.py`](https://github.com/dask/dask/blob/main/dask/bytes/core.py) commit with function `expand_paths_if_needed` defined.

Then commit https://github.com/dask/dask/commit/77d51a1ae3abfff6645132c7100911a37369d58c removes `expand_paths_if_needed` from [`dask/bytes/core.py`](https://github.com/dask/dask/blob/main/dask/bytes/core.py), and instead relies on the code copied by https://github.com/fsspec/filesystem_spec/commit/e34867fcd32ce057397c5572124d5bd6de8ee714 to [`fsspec/core.py`](https://github.com/fsspec/filesystem_spec/blob/master/fsspec/core.py).

Reply:
I'm not sure how relevant the history is any more, but I suppose we can maintain the link to old code.

Reply:
Oh, it's actually a link to the new code. I'm using a permalink (with commit) to the current code in case of changes in the future.